src/shared/components/mutationMapper/MutationMapper.tsx

```diff
@@ -479,6 +481,14 @@ export default class MutationMapper<
             this.store.dataStore.applyFilter
         );
 
+        return sortedFilteredData;
+    }
+
+    @computed
+    protected get mutationsGroupedByProteinImpactType() {
+        let sortedFilteredData = this
+            .sortedFilteredDataWithoutProteinImpactTypeFilter;
+
         // also apply lazy mobx table search filter
         sortedFilteredData = sortedFilteredData.filter(m =>
```

Comment:
Might be better to define this as a `const`

```suggestion
        // also apply lazy mobx table search filter
        const sortedFilteredData = this
            .sortedFilteredDataWithoutProteinImpactTypeFilter.filter(m =>
```

Reply:
Changed it to a const.